.github/workflows/torch-inductor/scripts/run_torchinductor_acc.sh

```diff
@@ -10,6 +10,15 @@ source /tmp/torchinductor_venv/bin/activate
 # shellcheck source=/dev/null
 source "$INDUCTOR"/scripts/common.sh
 
+# Dependency of 'torch/fx/experimental/validator.py'.
+pip3 install --upgrade z3-solver
+
+# Build our own triton.
+pip3 uninstall pytorch-triton -y
+cd $ROOT/python || exit
+rm -rf build
+pip3 install -e .
```

Comment:
Why do you need to build it again? 

Isn't it done in `install_triton.sh`?

Reply:
Fixed and cleaned up other dup code related to this across the different scripts.